src/wolfgame.py

```diff
@@ -7759,3 +7754,4 @@ def frole(cli, nick, chan, rest):
             del COMMANDS[comd]
 
 # vim: set sw=4 expandtab:
+
```

Comment:
No clue. I know I didn't manually put it there, so my guess is funny line ending shenanigans.